package-hash.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
 const packageHash = require('package-hash');
+const determineModulesPlugin = require('./plugins/modules');
 
-const plugins = require('./plugins/best-match')
+const plugins = Object.keys(determineModulesPlugin.supported)
+	.map(k => determineModulesPlugin.supported[k])
```

Comment:
Just starts with `[babel-plugin-transform-es2015-modules-commonjs]` and run `.concat(require('./plugins/best-match')` on that.